sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosItemRequestOptions.java

```diff
@@ -203,6 +204,51 @@ public CosmosItemRequestOptions setSessionToken(String sessionToken) {
         return this;
     }
 
+    /**
+     * Gets the boolean to only return the headers and status code in Cosmos DB response
+     * in case of Create, Update and Delete operations on CosmosItem.
+     *
+     * If set to false, this removes the resource from response. It reduces networking
+     * and CPU load by not sending the resource back over the network and serializing it on the client.
+     *
+     * This feature does not impact RU usage for read or write operations.
+     *
+     * NOTE: This flag is also present on {@link com.azure.cosmos.CosmosClientBuilder},
+     * however if specified on {@link CosmosItemRequestOptions},
+     * it will override the value specified in {@link com.azure.cosmos.CosmosClientBuilder} for this request.
+     *
+     * By-default, this is null.
+     *
+     * @return a boolean indicating whether resource will be included in the response or not for this request.
+     */
+    public Boolean isContentResponseOnWriteEnabled() {
+        return contentResponseOnWriteEnabled;
+    }
+
+    /**
+     * Sets the boolean to only return the headers and status code in Cosmos DB response
+     * in case of Create, Update and Delete operations on CosmosItem.
+     *
+     * If set to false, this removes the resource from response. It reduces networking
```

Comment:
two comments:
1. previously we had the notion of document/resource. Now there is not resource notion anywhere in our api or javadoc.
I would change "resource" to payload or data item.
2. "this removes the resource from response". Instead of saying "remove", I would reword this to say service doesn't return the payload. as it may become confusing what "removing" means.

same for the setter api.

Reply:
Good points, will make these changes across javadocs and will also check for CFP.

Reply:
@moderakh - updated the java docs.